client/app/dedicatedCloud/dashboard/legacy/controller.js

```diff
@@ -0,0 +1,66 @@
+import _ from 'lodash';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $scope,
+    $state,
+    $stateParams,
+    $timeout,
+    $translate,
+    $uibModal,
```

Comment:
I've never done it, but this is a great idea !

Reply:
@marie-j  I was not able to locate `@ovh-ux/ng-ui-router-layout` or anything that looks like it, are you sure about the name ?

Reply:
`ng-uirouter-layout` (named as is actually) has been moved

from (`ovh-manager-dedicated`): `https://github.com/ovh-ux/ovh-manager-dedicated/blob/master/client/app/components/angular-ui-router/angular-ui-router.module.js`

to (`manager`):  `https://github.com/ovh-ux/manager/blob/master/packages/components/ng-uirouter-layout/src/index.js`.

Hope it can help you:)